django_cloud_deploy/cli/prompt.py

```diff
@@ -1160,9 +1215,12 @@ def prompt(self, command: Command, console: io.IO,
         step_template = '<b>[{}/{}]</b>'
         first_step = step_template.format(1, total_steps)
 
-        creds = self._get_creds(console, first_step, args)
+        auth_client = auth.AuthClient()
+        creds = self._get_creds(console, first_step, args, auth_client)
+        active_account = auth_client.get_active_account()
```

Comment:
auth_client.get_active_account() will return an empty string or None (I am not sure whether it is None or empty string in python) if the user has not logged in with gcloud. Will this be a problem?

Reply:
it returns an empty string I confirmed this manually. but I can add a check to make sure it doesnt crash.

Reply:
There was a check already asserting it is not an empty string, as the user must have been logged in by that point.